eslint.config.mjs

```diff
@@ -70,6 +70,15 @@ export default tseslint.config(
       "@typescript-eslint/no-unused-vars": "off",
       "no-console": "error",
       "@typescript-eslint/no-unsafe-argument": "off",
+
+      "@typescript-eslint/prefer-destructuring": [
+        "error",
+        { array: true, object: true },
+        {
+          enforceForRenamedProperties: false,
+          enforceForDeclarationWithTypeAnnotation: false,
+        },
+      ],
     },
```

Comment:
There's a few weird situations from this new rule (like `({ left: this.x, top: this.y } = bbox);`) , but I still count it as a net positive.

Reply:
I didn't know that `enforceForRenamedProperties` defaults to true in eslint-config-love, so was trying to fix all the errors including those for renamed properties at first. I seem to have forgotten to revert the changes made at that time when I disabled that option later.